applications/luci-app-crowdsec-firewall-bouncer/htdocs/luci-static/resources/view/crowdsec-firewall-bouncer/form.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+'require view';
+'require form';
+'require tools.widgets as widgets';
+
+return view.extend({
+	render: function() {
+		var m, s, o;
+
+		m = new form.Map('crowdsec', _('CrowdSec'),
+			_('Gain <a href="http://www.crowdsec.net">crowd-sourced</a> protection against malicious IPs. ' +
+			'Benefit from the most accurate CTI in the world.'));
+
+
+		s = m.section(form.TypedSection, 'bouncer', _('Bouncer'));
+		s.anonymous = true;
+
+		o = s.option(form.Flag, 'enabled', _('Enable'));
+		o.default = '0';
+		o.rmempty = false;
+
+		o = s.option(form.Value, 'api_url', _('URL of local API'),
+			_('The URL of your local CrowdSec API instance.')); 
+		o.default = '';
+		o.rmempty = false;
+
+		o = s.option(form.Value, 'api_key', _('API key'),
+			_('The key of your bouncer as registered on the local CrowdSec API.'));
+		o.default = '';
+		o.password = true;
+		o.rmempty = false;
+
+		o = s.option(widgets.DeviceSelect, 'interface', _('Filtered interfaces'),
+			_('List of interfaces with traffic to be filtered.'));
+		o.modalonly = true;
```

Comment:
Just a minor remark, you can drop the `modalonly` property, it is only meaningful when the option is nested within `GridSection` sections.